docs/user-guide/10-minutes-to-awkward-array.md

```diff
@@ -85,71 +85,65 @@ In order to plot the taxi routes, we can use the waypoints given by the `path` f
 
 The fields of a record can be accessed with attribute notation:
 
-```{code-cell} ipython3
+```{code-cell} python3
 taxi.trip
 ```
 
 or using subscript notation:
 
-```{code-cell} ipython3
+```{code-cell} python3
 taxi["trip"]
 ```
 
 Field lookup can be nested, e.g. with attribute notation:
 
-```{code-cell} ipython3
+```{code-cell} python3
 taxi.trip.path
 ```
 
 or with subscript notation:
 
-```{code-cell} ipython3
+```{code-cell} python3
 taxi["trip", "path"]
 ```
 
 Let's look at two fields of interest, `path.latdiff`, and `begin.lat`, and their types:
 
-```{code-cell} ipython3
+```{code-cell} python3
 taxi.trip.path.latdiff
 ```
 
-```{code-cell} ipython3
+```{code-cell} python3
 taxi.trip.path.latdiff.type.show()
 ```
 
-```{code-cell} ipython3
+```{code-cell} python3
 taxi.trip.begin.lat
 ```
 
-```{code-cell} ipython3
+```{code-cell} python3
 taxi.trip.begin.lat.type.show()
 ```
 
 Clearly, these two arrays have _different_ dimensions. When we add them together, we want the operation to broadcast: each trip has one starting point, but multiple waypoints. In NumPy, broadcasting aligns *to the right*, which means that arrays with differing dimensions are made compatible against one another by adding length-1 dimensions to the front of the shape:
 
-```{code-cell} ipython3
+```{code-cell} python3
 x = np.array([1, 2, 3])
-y = np.array([
-    [4, 5, 6],
-    [7, 8, 9]
-])
+y = np.array([[4, 5, 6], [7, 8, 9]])
 np.broadcast_arrays(x, y)
 ```
 
 In Awkward, broadcasting aligns *to the left* by default, which means that length-1 dimensions are added *to the end* of the shape:
 
-```{code-cell} ipython3
-x = ak.Array([1, 2]) # note the missing 3!
-y = ak.Array([
-    [4, 5, 6],
-    [7, 8, 9]
-])
+```{code-cell} python3
+x = ak.Array([1, 2])  # note the missing 3!
+y = ak.Array([[4, 5, 6], [7, 8, 9]])
 ak.broadcast_arrays(x, y)
```

Comment:
I think you applied an automated code-formatter here, but the usefulness of this formatting was to explain broadcasting geometrically. It's not wrong to have them adhere to standard formatting here, but it's pedagogically _better_ for them to have the special formatting.